plugins/lookup/filetree.py

```diff
@@ -31,7 +31,7 @@
 - name: Template files (explicitly skip directories in order to use the 'src' attribute)
   ansible.builtin.template:
     src: '{{ item.src }}'
-    dest: /web/{{ item.path }}
+    dest: /web/{{ item.path | splitext | first }}
```

Comment:
Can you add a comment to explain this? I think it would be helpful to users to know why `| splitext | first` are here.

Reply:
Sure thing.

Reply:
I'm not sure how well my explanation works but there's something there now :)

Reply:
Looks good :)